py/sdk/v2/kg.py

```diff
@@ -103,39 +103,39 @@ async def get_entities(
 
         return await self._make_request("GET", "entities", params=params)  # type: ignore
 
-    async def get_triples(
+    async def get_relationships(
```

Comment:
Let's keep v2 method names, it's fine to change the internal naming conventions, though